website/docs/user_guides/run_chaos_experiment.md

```diff
@@ -68,16 +68,18 @@ kubectl delete -f pod-failure-example.yaml
 
 ### Watch your chaos experiments in Chaos Dashboard
 
-Chaos Dashboard is currently only available for TiDB clusters. Stay tuned for more supports or join us in making it happen.
+Chaos Dashboard is a Web UI for managing, designing, monitoring Chaos Experiments. Stay tuned for more supports or join us in making it happen.
 
 > **Note:**
->
+> 
 > If Chaos Dashboard was not installed, upgrade Chaos Mesh by executing `helm upgrade chaos-mesh helm/chaos-mesh --namespace=chaos-testing --set dashboard.create=true`.
 
 A typical way to access it is to use `kubectl port-forward`:
 
 ```bash
-kubectl port-forward -n chaos-testing svc/chaos-dashboard 8080:80
+kubectl port-forward -n chaos-testing svc/chaos-mesh-chaos-dashboard 2333:2333
```

Comment:
The service name is kind of redundant. Do you think it is okay to use `chaos-dashboard` only? We can define the name in template